nilearn/plotting/surface/_matplotlib_backend.py

```diff
@@ -302,3 +313,170 @@ def _threshold_and_rescale(data, threshold, vmin, vmax):
     """
     data_copy, vmin, vmax = _rescale(data, vmin, vmax)
     return data_copy, _threshold(data, threshold, vmin, vmax), vmin, vmax
+
+
+class MatplotlibBackend(SurfaceBackend):
+    def _check_backend_params(self, params):
+        pass
```

Comment:
This one does not check any parameters for the moment, as the actual code has only checks for plotly but not matplotlib. But I plan to put here `symmettic_cmap` and `title_font_size`, as they are not used by matplotlib backend.